odoo/addons/base/models/ir_ui_view.py

```diff
@@ -555,25 +570,33 @@ def get_inheriting_views_arch(self, view_id, model):
                 for view in views.sudo()
                 if not view.groups_id or (view.groups_id & user_groups)]
 
-    @api.model
-    def raise_view_error(self, message, view_id):
-        view = self.browse(view_id)
-        not_avail = _('n/a')
-        message = (
-            "%(msg)s\n\n" +
-            _("Error context:\nView `%(view_name)s`") +
-            "\n[view_id: %(viewid)s, xml_id: %(xmlid)s, "
-            "model: %(model)s, parent_id: %(parent)s]"
-        ) % {
-            'view_name': view.name or not_avail,
-            'viewid': view_id or not_avail,
-            'xmlid': view.xml_id or not_avail,
-            'model': view.model or not_avail,
-            'parent': view.inherit_id.id or not_avail,
-            'msg': message,
-        }
-        _logger.info(message)
-        raise ValueError(message)
+    def raise_view_error(self, message):
```

Comment:
Dont change the call of raise_view_error juste raise here using the view_id received.

Reply:
the idea was to make a PR for master and another for stable with minimal modifications, but OK